pkg/operator/ceph/csi/spec.go

```diff
@@ -511,6 +557,15 @@ func createCSIDriverInfo(clientset kubernetes.Interface, name string, ownerRef *
 	return err
 }
 
+// deleteCSIDriverInfo deletes CSIDriverInfo and returns the error if any
+func deleteCSIDriverInfo(clientset kubernetes.Interface, name string) error {
+	err := clientset.StorageV1beta1().CSIDrivers().Delete(name, &metav1.DeleteOptions{})
+	if err != nil {
```

Comment:
Instead of just logging the error, I'm suggesting something like this:
```
if err != nil && errors.IsNotFound(err) {
    //. no reason to fail if the driver is not found
    return nil
}
return nil
```
The same pattern would apply to the `DeleteService()` func.